tox.ini

```diff
@@ -26,7 +26,7 @@ commands =
     coverage erase
     coverage run --parallel --source={envsitepackagesdir}/pytest_check,tests -m pytest tests
     coverage combine
-    coverage report
+    coverage report --fail-under=100
```

Comment:
I think this simple change will allow you to automatically enforce your desire for 100% code coverage?